glusterd2/commands/volumes/volume-create.go

```diff
@@ -42,6 +43,10 @@ func validateVolCreateReq(req *api.VolCreateReq) error {
 		return gderrors.ErrInvalidVolName
 	}
 
+	if gutils.IsReservedKeyword(req.Name) {
```

Comment:
please add a e2e test case of this

Reply:
Same as mentioned for other case, please add this as well in the same issue

Reply:
these code changes are in this PR so that I would like to see an e2e in this PR itself.